### DISCUSSION TITLE
Apple Music have overlapping titles

### DISCUSSION THREAD

**Discussion by RSickenberg** [lowlighter/metrics#864] [Q&A] [answered]
Hello, first of all, thank you for this masterpiece !

I have an issue with the Plugin music that overflow the titles on the SVG's, do you know if I need to configure something?

Regards

![image](https://user-images.githubusercontent.com/16922548/153161108-240b4582-bda6-4908-9cf5-404ac0befd19.png)
> _Source: https://github.com/RSickenberg_

**Comment by lowlighter** [comment, marked as answer]
Hi 👋 !

Thanks for your support 🙂 

It seems to be a css issue which should be fixed upstream, there's no extra settings to be configured.

However you can use [`extras_css`](https://github.com/lowlighter/metrics/blob/master/source/plugins/core/README.md#-custom-css-styling) as a workaround to fix it while waiting for a fix: 
```yaml
extras_css: |
  .track .name { max-width: 175px !important; }
```

May need to tweak a bit though, I put 175px as an example but there might be better a value somewhere else

**Reply by RSickenberg** [reply]
Working like a glove! Let me know if you want a PR for this :-)

**Reply by lowlighter** [reply]
Nice!
Yes sure, you can open a PR and link #866 to it 👍

**Reply by RSickenberg** [reply]
I need to find the time but maybe this week is ok for me.

**Reply by lowlighter** [reply]
No worries 👍

**Reply by RSickenberg** [reply]
@lowlighter Done 😄, let me know if it fits the needs

**Reply by lowlighter** [reply]
Yes it's good,, thanks a lot!

**Comment by RSickenberg** [comment]
Hello @lowlighter, I tested it after the release and the issue is still present unfortunately :(
![image](https://user-images.githubusercontent.com/16922548/156017386-bdf6251c-c844-45c6-84ed-407663b4d2f2.png)

**Reply by lowlighter** [reply]
Ok I'll reopen the issue 👍
